pytest/tests/sanity/network_drop_package.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, *args, **kwargs):
 
     async def handle(self, msg, fr, to):
         if msg.enum == 'Block':
-            h = msg.Block.BlockV2.header.BlockHeaderV3.inner_lite.height
+            h = msg.Block.BlockV2.header.inner_lite().height
```

Comment:
do we need to do the same with Block and other structs? like `msg.Block.header().inner_lite().height`

Reply:
Ideally. But in this PR I only fixed the one that is broken atm.